SpacetimeDB/.github/workflows/tag-release.yml

```
@@ -0,0 +1,22 @@
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ vars.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+ 
+      - name: Retag the image 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)